src/main/java/edu/harvard/iq/dataverse/externaltools/ExternalToolServiceBean.java

```diff
@@ -131,6 +147,7 @@ public static ExternalTool parseAddExternalToolManifest(String manifest) {
         String displayName = getRequiredTopLevelField(jsonObject, DISPLAY_NAME);
         String description = getRequiredTopLevelField(jsonObject, DESCRIPTION);
         String typeUserInput = getRequiredTopLevelField(jsonObject, TYPE);
+        String scopeUserInput = getRequiredTopLevelField(jsonObject, SCOPE);
         String contentType = getOptionalTopLevelField(jsonObject, CONTENT_TYPE);
         //Legacy support - assume tool manifests without any mimetype are for tabular data
         if(contentType==null) {
```

Comment:
This is going to set a content type on dataset level tools

Reply:
Yes, this is a bug. We don't want dataset tools to have a content type. Good catch. Thanks.

Reply:
@qqmyers eventually we may use this - for example, show the dataset level explore only if the dataset has at least one file of content type X - but for now we're going with simple.

Reply:
If I understand @scolapasta's suggestion - we may have a use for the content type, with *some* dataset-level external tools; but it sounds like it still needs to be nullable in the table; and it still should be null by default. 
In other words, if we adopt this scheme, then we definitely DON'T want to set the type to tab-delimited by default.